src/edit_mode/vi/command.rs

```diff
@@ -306,7 +306,7 @@ impl Command {
                 Motion::LeftBefore(c) => {
                     Some(vec![ReedlineOption::Edit(EditCommand::CutLeftBefore(*c))])
                 }
-                Motion::Start => Some(vec![ReedlineOption::Edit(EditCommand::CutFromLineStart)]),
+                Motion::Start => Some(vec![ReedlineOption::Edit(EditCommand::CutFromStart)]),
```

Comment:
That leads to the wrong behavior (see my addendum on #459). Might have still been on your working tree.

Reply:
Oh yeah, I had some trouble initially merging your fixes into my working tree. I'll revert this